src/test/java/edu/kis/powp/jobs2d/events/FlipVerticalSecretCommandOptionListener.java

```diff
@@ -0,0 +1,78 @@
+package edu.kis.powp.jobs2d.events;
+
+import edu.kis.powp.jobs2d.command.*;
+import edu.kis.powp.jobs2d.command.manager.DriverCommandManager;
+import edu.kis.powp.jobs2d.features.CommandsFeature;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class FlipVerticalSecretCommandOptionListener implements ActionListener {
+    @Override
+    public void actionPerformed(ActionEvent e) {
+
+        DefaultCompoundCommand compound = null;
+        List<DriverCommand> commands = new ArrayList<DriverCommand>();
+        DriverCommand first, second, third, lastOne; // test DriverCommands
+        DriverCommandFlipVertical flipVisitor = null;
+
+        try {
+            commands.add(new SetPositionCommand(-20, -50));
+            commands.add(new OperateToCommand(-20, -50));
+            commands.add(new SetPositionCommand(-20, -40));
+            compound = new DefaultCompoundCommand(commands);
+            first = CommandsFeature.deepCopyCommand(compound);
+
+            commands.clear();
+            commands.add(new OperateToCommand(-20, 50));
+            commands.add(new SetPositionCommand(0, -50));
+            commands.add(new OperateToCommand(0, -50));
+            commands.add(new SetPositionCommand(0, -40));
+            commands.add(new OperateToCommand(0, 50));
+            compound = new DefaultCompoundCommand(commands);
```

Comment:
code redundancy in tests. It would be easier to create some simple reusable test factories.